server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -146,6 +152,14 @@ def _get_aggregate_function(
         self: Self, agg_function: "AggregateFn"
     ) -> functions.AggregateFunction:
         match agg_function:
+            # Commenting this since postgres and redshift
```

Comment:
By doing so, you forbid the use of these functions for translators where it's implemented no? 
Perhaps, you could update the match case based on the instance's Dialect.

Reply:
After checks, it was not in the list as you can see here : https://github.com/ToucanToco/weaverbird/blob/master/server/src/weaverbird/backends/pypika_translator/translators/base.py#L148
I just added this comment so that will keep in mind pypika has those built-in function and for next card adapt depending on the translator what works and what doesn't !

Reply:
:+1: